pkg/plugins/client.go

```diff
@@ -110,15 +110,19 @@ func (c *Client) SendFile(serviceMethod string, data io.Reader, ret interface{})
 }
 
 func (c *Client) callWithRetry(serviceMethod string, data io.Reader, retry bool) (io.ReadCloser, error) {
-	req, err := c.requestFactory.NewRequest(serviceMethod, data)
+	var retries int
+	start := time.Now()
+	savedData, err := ioutil.ReadAll(data)
```

Comment:
this is ugly I know - just hacking around the contentlength difference in the error above.. any other idea?